RPI/monitor.js

```diff
@@ -1,30 +1,75 @@
 const GrovePi = require('grovepi').GrovePi
 const Board = GrovePi.board
-const AirQualityAnalogSensor = GrovePi.sensors.AirQualityAnalog
+const AnalogSensor = GrovePi.sensors.base.Analog
 
-function start(onChange = console.log) {
+const DC_GAIN = 8.5;               // The DC gain of amplifier
```

Comment:
All these constants come from the [sample code](https://wiki.dfrobot.com/CO2_Sensor_SKU_SEN0159).